frontend/src/concepts/pipelines/content/utils.tsx

```diff
@@ -30,6 +31,9 @@ export type RunStatusDetails = {
 const UNKNOWN_ICON = <QuestionCircleIcon />;
 const UNKNOWN_STATUS = 'warning';
 
+export const getStorageState = (run?: PipelineRunKFv2 | null): StorageStateKF | undefined =>
+  run?.storage_state;
```

Comment:
nit: I don't think accessing storage_state off of run constitutes a helper function, where this is seemingly used in only 1 place as well.

Reply:
yeah does not need a helper